src/ui/sessionManager.ts

```diff
@@ -83,7 +83,11 @@ export class SessionManager implements vscode.DebugAdapterDescriptorFactory, Dis
     const session = new Session(debugSession);
     if (debugSession.configuration['__pendingTargetId']) {
       const pendingTargetId = debugSession.configuration['__pendingTargetId'] as string;
-      const target = this._pendingTarget.get(pendingTargetId)!;
+      const target = this._pendingTarget.get(pendingTargetId);
+      if (!target) {
```

Comment:
This worries me. Do you have a particular scenario for this? As I see, we only add to `this._pendingTarget` once, and remove from there once, and I am not sure about the situation which causes this issue.

Reply:
Another (possible future) direction I was looking for in this was being able to attach to _all_ browser tabs which matched the url or urlFilter. In that scenario I was able to get an error to be thrown here probably 20% of the time by quickly navigating between pages or opening/closing tables. I'm not able to repro in the current single-tab attachment though, so it's probably safe to remove this from this PR.